backend/infrahub/core/diff/combiner.py

```diff
@@ -295,17 +293,20 @@ def _combine_relationships(
                 combined_relationship_elements = self._combined_cardinality_many_relationship_elements(
                     earlier_elements=earlier_relationship.relationships, later_elements=later_relationship.relationships
                 )
-            combined_relationship = EnrichedDiffRelationship(
-                name=later_relationship.name,
-                label=later_relationship.label,
-                cardinality=later_relationship.cardinality,
-                changed_at=later_relationship.changed_at or earlier_relationship.changed_at,
-                action=self._combine_actions(earlier=earlier_relationship.action, later=later_relationship.action),
-                path_identifier=later_relationship.path_identifier,
-                relationships=combined_relationship_elements,
-                nodes=set(),
-            )
-            combined_relationships.add(combined_relationship)
+            combined_relationship_elements = {cre for cre in combined_relationship_elements if cre.properties}
+            includes_parent = self._child_parent_uuid_map.get(node_id, (None, None))[1] == later_relationship.name
```

Comment:
This part looks a bit hairy, might it be cleaner to have a method that takes the node_id as input and optionally returns the secondary value of the tuple if the node exists in `self._child_parent_uuid_map`?  "_get_parent_rel_name" or so?